.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+<!--
+Thanks for raising a Spring Integration issue. What sort of issue are you raising?
+
+Question
+
+Please ask questions about how to use something, or to understand why something isn't
+working as you expect it to, on Stack Overflow using the `spring-integration` tag.
+
+Bug report
+
+Please provide details of the problem, including the version of Spring Integration that you
+are using. If possible, please provide a test case or sample application that reproduces
+the problem. This makes it much easier for us to diagnose the problem and to verify that
+we have fixed it.
```

Comment:
How about to mention (and link) that we prefer JIRA instead of GH issues for this project?
Or do we not anymore?